static/css/wall_display.css

```
@@ -160,6 +160,55 @@ html {
+  border: 2px solid rgba(255, 255, 255, 0.5);
+  transition: all 0.3s ease;
+}
+.hub-indicator.active {
+  background-color: rgba(255, 255, 255, 0.9);
+  border-color: #fff;
+  box-shadow: 0 0 15px rgba(255, 255, 255, 0.8);
+}
+.hub-indicator.flashing {
+  animation: hubFlash 0.5s ease-in-out infinite;
+}
+@keyframes hubFlash {
```

Stylelint (17.4.0)

[error] Expected keyframe name "hubFlash" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)